api_tests/nodes/views/test_node_preprints.py

```diff
@@ -1,36 +1,35 @@
+import pytest
+
 from nose.tools import *  # flake8: noqa
```

Comment:
Is nose still being used? If so, can the functionality be extracted? Or was there more to convert here, I have vague memories of that.